front/src/styles/common.css

```diff
@@ -21,3 +21,6 @@ button.second-color {
 .main-content {
     min-height: calc(100vh - (var(--header-height) + var(--footer-height) + var(--footer-bottom-height)));
   }
+* {
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
is it the font from design? why do you add it?

Reply:
removed